esp/esdllib/README.md

```diff
@@ -0,0 +1,2142 @@
+# ESDL Scripting
+
```

Comment:
@asselitx Maybe we should put a "## History" title above this text and add a paragraph above simply describing how powerful ESDL scripting is in it's current state?